### DISCUSSION TITLE
Error running make; make install on PNETCDF

### DISCUSSION THREAD

**Discussion by Nickfu0911** [firedrakeproject/firedrake#3507] [Install FAQ] [answered]
Hi,

I encountered some problems when installing Firedrake on MacOS 14.4. The error message says:  "Error running make; make install on PNETCDF". I've attached my install log file below. Any help would be highly appreciated! (I'm using Python 3.11 from Homebrew)
[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/14999015/firedrake-install.log)

**Comment by Ig-dolci** [comment]
Could you please also add the `configure.log` file (located at `src/petsc`)?

**Reply by Nickfu0911** [reply]
[configure.log](https://github.com/firedrakeproject/firedrake/files/14999283/configure.log)
Sure, here it is

**Comment by Ig-dolci** [comment]
It looks like be associated with a failure to find `m4`. Can you please execute `brew install m4` and try the firedrake install again?

**Comment by Nickfu0911** [comment]
<img width="465" alt="Screenshot 2024-04-16 at 17 25 05" src="https://github.com/firedrakeproject/firedrake/assets/167223389/79846c93-ea67-41d3-882b-39095695fd8c">
I checked that I already had m4 installed in homebrew. However, a pop-up window did appear when installing Firedrake (I attached the screenshot above), asking me to install m4 every time, although I already have it installed.

**Reply by dham** [reply, marked as answer]
I hit this yesterday. It looks like apple have made a mess of this. It appears command line tools has renamed m4 to gm4 (for GNU m4) but MacOS itself hasn't caught up, so `/usr/bin/m4` goes looking for `m4` in command-line tools and fails. It then fires off that pop-up. This is very stupid.

The solution that works for me is to make a symlink somewhere in my $PATH to map m4 to /Library/Developer/CommandLineTools/usr/bin/gm4

**Reply by Nickfu0911** [reply]
Many thanks! It works now.
